bitnami/grafana/Chart.yaml

```diff
@@ -33,4 +31,4 @@ maintainers:
 name: grafana
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/grafana
-version: 9.3.2
+version: 9.4.0
```

Comment:
Not sure about the version. I think is better a major change and a quick note in the [Upgrading section](https://github.com/bitnami/charts/blob/b24c42383a0c240963afed642c5a6a0b23ad3c7f/bitnami/grafana/README.md#upgrading) Why? Users with `imageRenderer.enabled=true` could face issues because their `imageRender` service and deployment will be remove during the upgrade.

Reply:
OK, no problem with that approach 👍

Reply:
see https://github.com/bitnami/charts/pull/20377/commits/128b9a1f268221765d7abaf0b364d5d0b9fe252a